frontend/src/home.html

```diff
@@ -4,15 +4,31 @@
 <head>
   <meta charset="UTF-8">
   <link rel="stylesheet" href="./style.css">
-
+  <style>
+    .logout-link {
+      display: inline-block;
+      position: absolute;
+      top: 280px;
+      right: 220px;
+      color: #ffffff;
+      text-decoration: none;
+    }
+  </style>
```

Comment:
You should add this styling to the `.css` file, maybe we could reuse this css class